azure-container-apps/README.md

```diff
@@ -0,0 +1,371 @@
+# Deploy 1Password SCIM Bridge on Azure Container Apps using the Azure Portal
+
+_Learn how to deploy 1Password SCIM Bridge on the [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps/#overview) service._
+
+This deployment consists of two [containers](https://learn.microsoft.com/en-us/azure/container-apps/containers): One for the SCIM bridge and another for Redis. There's also an [ingress](https://learn.microsoft.com/en-us/azure/container-apps/ingress-overview) for the SCIM bridge container. There are a few benefits to deploying 1Password SCIM Bridge on Azure Container Apps:
+
+- For standard deployments, the service will host your SCIM bridge for ~$16 USD/month (as of January 2024). Container Apps pricing is variable based on activity, and you can learn more on [Microsoft's pricing page](https://azure.microsoft.com/en-us/pricing/details/container-apps/).
+- You don't need to manage a DNS record. Azure Container Apps automatically provides a unique one for your SCIM bridge domain.
+- Azure Container Apps automatically handles TLS certificate management on your behalf.
+- The SCIM bridge can be deployed directly to Azure from the Portal using this guide or via the Azure Shell or command line tools in your local terminal using the [support guide](https://support.1password.com/scim-deploy-azure/). There is no requirement to clone this repository.
+
+**Table of contents:**
+
+- [Before you begin](#before-you-begin)
+- [Step 1: Configure the `scimsession` credentials](#step-1-configure-the-scimsession-credentials)
+- [Step 2: Create the container app](#step-2-create-the-container-app)
+- [Step 3: Configure and deploy your SCIM bridge](#step-3-configure-and-deploy-your-scim-bridge)
+- [Step 4: Connect your identity provider](#step-4-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Resource recommendations](#appendix-resource-recommendations)
+- [Get help](#get-help)
+
+## Before you begin
+
+Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there. You'll also need the following:
```

Comment:
That suggestion looks good to me. I'll also apply it to the other files.

For the preparation.md file, I can look into updating that in the future once we're done with this project and the DigitalOcean equivalent of it – I don't want too many changes to a bunch of files in this PR 😄

Reply:
> That suggestion looks good to me. I'll also apply it to the other files.

Great!

> For the preparation.md file, I can look into updating that in the future once we're done with this project and the DigitalOcean equivalent of it – I don't want too many changes to a bunch of files in this PR 😄

Yes, agreed. :) I tried to highlight any thoughts that went beyond this PR as out of scope, but I figured it would be worthwhile mentioning them in context here (and so I wouldn't forget later).